services/QuillLMS/spec/support/shared_contexts/stripe_integration/stripe_subscription.rb

```diff
@@ -35,7 +37,7 @@
       discount: nil,
       ended_at: nil,
       items: {
-        object: 'list',
+        object: 'lisk',
```

Comment:
Not sure if this matters, but is this a typo?

Reply:
Whoops.  Vim navigation typo.